src/components/projects/create-wp-form/create-wp-form.tsx

```diff
@@ -75,20 +93,76 @@ const CreateWPForm: React.FC = () => {
     }
   };
 
-  const handleSubmit = (e: any) => {
+  const states = {
+    name: setName,
+    wbsNum: setWbsNum,
+    crId: setCrId,
+    startDate: setStartDate,
+    duration: setDuration
+  };
+
+  const handleSubmit = async (e: any) => {
     e.preventDefault();
 
     // exits handleSubmit if form input invalid (should be changed in wire up)
-    const wbsNum: WbsNumber = validateWBS(e.target.wbsNum.value.trim());
+    let wbsNum: WbsNumber;
+    try {
+      wbsNum = validateWBS(projectWbsNum);
 
-    if (!isProject(wbsNum!)) {
-      alert('Please enter a valid Project WBS Number.');
-      return;
+      const { userId } = auth.user!;
+
+      if (!isProject(wbsNum!)) {
+        alert('Please enter a valid Project WBS Number.');
+        return;
+      }
+      const depWbsNums = dependencies.map((dependency) => {
+        const depWbsNum = validateWBS(dependency.trim());
+        return {
+          carNumber: depWbsNum.car,
+          projectNumber: depWbsNum.project,
+          workPackageNumber: depWbsNum.workPackage
+        };
+      });
+      await mutateAsync({
+        userId,
+        name: name.trim(),
+        crId,
+        projectWbsNum: {
+          carNumber: wbsNum.car,
+          projectNumber: wbsNum.project,
+          workPackageNumber: wbsNum.workPackage
+        },
+        startDate,
+        duration,
+        dependencies: depWbsNums,
+        expectedActivities,
+        deliverables
+      });
+      history.push(routes.CHANGE_REQUESTS);
+    } catch (e) {
+      console.log(e);
+      alert('wbs num not valid');
```

Comment:
```suggestion
      alert('something went wrong');
```

Reply:
this is because this catches any errors from the back-end call and so it won't always be because of the wbs nums